src/portal_visualization/builders/imaging_builders.py

```diff
@@ -93,7 +93,7 @@ def get_conf_cells(self, **kwargs):
         conf = vc.to_dict()
         # Don't want to render all layers
         del conf["datasets"][0]["files"][0]["options"]["renderLayers"]
-        return get_conf_cells(conf)
+        return [VitessceConfig.from_dict(conf)]
```

Comment:
Another change for the sake of consistency: `get_configs` will now always return a list of configuration objects, instead of returning dicts for some classes, objects for others, and lists in a couple cases.